tomcat/src/main/java/com/techcourse/StreamToStringConverter.java

```diff
@@ -0,0 +1,32 @@
+package com.techcourse;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class StreamToStringConverter {
+    public static List<String> convertToLines(InputStream inputStream) {
+        List<String> ret = new ArrayList<>();
+        try {
```

Comment:
```suggestion
    try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream)))
```

try 문 안으로, BufferedReader 를 넣어서 try-with-resoucres 를 사용해도 괜찮을 거 같습니다!
( 직접 close 할 필요 X + 까먹을 우려 X 🫡)